001-finding_resources.md

```diff
@@ -0,0 +1,120 @@
+# Design Document 001 - Finding Resources
+
+TODO: Overview
+
+## Finding SDF URI Resources
+
+The following sections discuss how `<uri>` elements in SDF files are
+resovled in order to find meshes and models. This section does not cover
+finding other resources, such as plugins.
+
+### Current Process
+
+This section describes how Ignition Gazebo currently finds Mesh URI
+resources. SDF contains multiple instances of URI elements, but the current
+implementation of Gazebo handles only mesh URIs. The information contained
+here can be extrapolated to handle other types of resources, such as
+textures and audio files.
+
+A user of Ignition may not need or use Gazebo. If this describes you, then
```

Comment:
I think this part can be a bit confusing. The first time I read it I thought `Gazebo` meant Gazebo-classic. I think it's worth it spelling out "Ignition Gazebo".

Reply:
Thanks, addressed in a3696f8e717265fb69c906bc630f83723b50d8dd